src/Entity/LineItem.php

```diff
@@ -137,7 +137,7 @@ public function isAvailableForDate(DateTimeInterface $date): bool
         return $this->startAt <= $date && $this->endAt >= $date;
     }
 
-    public function getMaxAttempts(): int
+    public function getMaxAttempts(): ?int
```

Comment:
Maybe it could return a default value (0,1) if not set? Not sure it makes more sense do it here or in a later point when using it. What do you think?